Library/OcAudioLib/OcAudioVoiceOver.c

```diff
@@ -27,6 +27,68 @@
 
 #include "OcAudioInternal.h"
 
+//
+// Convert from Apple file id to file base name and type.
+//
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileVoiceOverOn = {
+  APPLE_VOICE_OVER_AUDIO_FILE_VOICE_OVER_ON,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileVoiceOverOff = {
+  APPLE_VOICE_OVER_AUDIO_FILE_VOICE_OVER_OFF,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileUsername = {
+  APPLE_VOICE_OVER_AUDIO_FILE_USERNAME,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFilePassword = {
+  APPLE_VOICE_OVER_AUDIO_FILE_PASSWORD,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileUsernameOrPasswordIncorrect = {
+  APPLE_VOICE_OVER_AUDIO_FILE_USERNAME_OR_PASSWORD_INCORRECT,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileAccountLockedTryLater = {
+  APPLE_VOICE_OVER_AUDIO_FILE_ACCOUNT_LOCKED_TRY_LATER,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileAccountLocked = {
+  APPLE_VOICE_OVER_AUDIO_FILE_ACCOUNT_LOCKED,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileVoiceOverBoot = {
+  OC_VOICE_OVER_AUDIO_FILE_VOICE_OVER_BOOT,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_OPEN_CORE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileVoiceOverBoot2 = {
+  APPLE_VOICE_OVER_AUDIO_FILE_VOICE_OVER_BOOT,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileClick = {
+  APPLE_VOICE_OVER_AUDIO_FILE_CLICK,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+STATIC APPLE_VOICE_OVER_FILE_MAP  mAppleVoiceOverAudioFileBeep = {
+  APPLE_VOICE_OVER_AUDIO_FILE_BEEP,
+  OC_VOICE_OVER_AUDIO_BASE_TYPE_APPLE
+};
+
+CONST APPLE_VOICE_OVER_FILE_MAP  *gAppleVoiceOverAudioFiles[AppleVoiceOverAudioFileMax] = {
+  [AppleVoiceOverAudioFileVoiceOverOn]                 = &mAppleVoiceOverAudioFileVoiceOverOn,
```

Comment:
Any big reason not to have an array of structures rather than pointers to structures?

Reply:
Only saving a few bytes. Switch to array of structures?

Reply:
Changed.